builtin/logical/mssql/path_config_lease.go

```diff
@@ -0,0 +1,103 @@
+package mssql
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigLease(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/lease",
+		Fields: map[string]*framework.FieldSchema{
+			"lease": &framework.FieldSchema{
```

Comment:
Please use `ttl` and `ttl_max`. We've moved away from the overloaded usage of `lease` to mean both the lease itself and the duration.